Added swedish language

MickeyMik opened the pull request.

MickeyMik committed:
Update langs.md

MickeyMik committed:
Update translations.js

Added Swedish translation

MickeyMik committed:
Create sv.html

MickeyMik committed:
Create instructions_sv.md

MickeyMik committed:
Merge pull request #1 from MickeyMik/patch-1

Update langs.md

MickeyMik committed:
Merge pull request #2 from MickeyMik/patch-2

Update translations.js

MickeyMik committed:
Merge pull request #3 from MickeyMik/patch-4

Create sv.html

MickeyMik committed:
Merge pull request #4 from MickeyMik/patch-5

Create instructions_sv.md

MickeyMik committed:
Add files via upload

MickeyMik committed:
Update langs.md

MickeyMik committed:
Add files via upload

MickeyMik committed:
Delete sv.png

MickeyMik committed:
Add files via upload

MickeyMik committed:
Delete sv.png

MickeyMik committed:
Add files via upload

MickeyMik committed:
Update README.md

Added credits and links, typo fixes.

MickeyMik committed:
Update README.md

MickeyMik committed:
typo fix

MickeyMik committed:
Removed Gtag, added contributors, update.

Based on IHTFY's commits

MickeyMik committed:
Restructure, added card color change, removed gtag

MickeyMik committed:
reformat, removed gtag, resized UI

MickeyMik committed:
reformat, Resize UI

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update draw.js

MickeyMik committed:
Update draw.js

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update pt-br.html

MickeyMik committed:
Update pt-br.html

MickeyMik committed:
Update pt-br.html

MickeyMik committed:
Update pt-br.html

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update board.js

MickeyMik committed:
Update styles.css

MickeyMik committed:
Update styles.css

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update pt-br.html

MickeyMik committed:
Update pt-br.html

MickeyMik committed:
Update pt-br.html

MickeyMik committed:
Update ca.html

MickeyMik committed:
Update cn.html

MickeyMik committed:
Update de.html

gjeuken commented:
Hi,
Thanks a lot for the Swedish translation! I really appreciate the help.
However, it looks like most of the (later) commits on this PR are unrelated to the translation. Would you be willing to reorganize the pull request? 

Thanks again

MickeyMik committed:
Clearboard() resets guess marker

MickeyMik committed:
Update board.js

MickeyMik committed:
Removed "button_clear()"

Uneccessary as clearboard does the exact same function, as it also resets the guess marker.

MickeyMik committed:
Update index.html

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update de.html

MickeyMik committed:
Update ca.html

MickeyMik committed:
Update cn.html

MickeyMik committed:
Update dk.html

MickeyMik committed:
Update es.html

MickeyMik committed:
fixed typo

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update fr.html

MickeyMik committed:
Update hu.html

MickeyMik committed:
Update it.html

MickeyMik committed:
Update nl.html

MickeyMik committed:
Update index.html

MickeyMik committed:
Update ca.html

MickeyMik committed:
Update sv.html

MickeyMik committed:
Update pt-br.html

MickeyMik committed:
Update nl.html

MickeyMik committed:
Update it.html

MickeyMik committed:
Update hu.html

MickeyMik committed:
Update fr.html

MickeyMik committed:
Update es.html

MickeyMik committed:
Update dk.html

MickeyMik committed:
Update de.html

MickeyMik committed:
Update cn.html

MickeyMik commented:
Yes, apologies, I'm new to how github works so I didn't know all of my changes would be added to the request.
The changes I've done are the ones IHTFY did in their fork, resized the UI, changed the graphics of the cards, and removed the Google analytics.
I'll change the pull request to only include the added language.
